frinx-uniconfig/user-guide/operational-procedures/uniconfig-properties/readme.md

```diff
@@ -0,0 +1,89 @@
+# Uniconfig Properties
+
+## Introduction
+
+UniConfig has input parameters to configure it precisely.
+All these parameters are located in the *application.properties* file.
+These parameters are divided into 2 groups - immutable parameters, 
+which are the majority, and mutable parameters, which can be
+changed in runtime without restarting UniConfig with *UniConfig Cloud Config*.
+
+Mutable properties consist of *crypto*, *schema-settings*, *callbacks*,
+*notifications.kafka*, *netconf-default-parameters*, *gnmi-default-parameters*
+and *cli-default-parameters* property prefixes. These properties are stored
+in the database in the table *properties* and are known as *default properties*.
+User can read *default properties* with *read-properties RPC* and update it with *update-properties RPC*.
+
+!!!
+After starting UniConfig, if the database contains these properties, 
+UniConfig will use the values of these properties from the database. But if these 
+parameters are not in the database, then the values from first UniConfig instance 
+right after its startup will be used (by application.properties file or env variables) 
+and saved to the database for the next UniConfig instances.
+!!!
+
+## UniConfig Cloud Config
+
+UniConfig Cloud Config is pretty the same technology as Spring Cloud Config with JDBC backend:
```

Comment:
grammar

`pretty the same` 

could be `pretty much the same` 

you should copy paste this test into e.g. google docs and it would highlight grammar issues like this one

Reply:
Would be nice to have a couple sentences in this section about what the "cloud config" is and why it is used .. in simple terms ... without having to read the spring cloud config docs (that are not super clear).

Reply:
done